CHANGELOG.unreleased.md

```diff
@@ -12,6 +12,7 @@
 - [Plugin] Disable test plugin action when the plugin is not loaded (PR [#5038](https://github.com/vatesfr/xen-orchestra/pull/5038))
 - [Home/Template] Ability to copy/clone VM templates [#4734](https://github.com/vatesfr/xen-orchestra/issues/4734) (PR [#5006](https://github.com/vatesfr/xen-orchestra/pull/5006))
 - [VM/bulk copy] Add fast clone option (PR [#5006](https://github.com/vatesfr/xen-orchestra/pull/5006))
+- [Audit] Back up fingerprints [#4844](https://github.com/vatesfr/xen-orchestra/issues/4844) (PR [#5077](https://github.com/vatesfr/xen-orchestra/pull/5077))
```

Comment:
Not clear enough, remember you are talking to users that did not work on the audit log.

Reply:
What's unclear in my sentence? it's the fingerprints notion or the back up notion?

Reply:
Both, we need figure out a way to make it clear to users what this feature does and why it's important.